Gems/GenAIFramework/Code/Include/GenAIFramework/Feature/JsonConversionUtils.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <GenAIFramework/Communication/AIModelRequestBus.h>
+
+#include <AzCore/JSON/document.h>
+#include <AzCore/JSON/rapidjson.h>
+#include <AzCore/JSON/writer.h>
+#include <rapidjson/rapidjson.h>
+
+namespace GenAIFramework::Internal
```

Comment:
I would use a more meaningful name for the namespace. E.g. `GenAIFramework::JsonUtils` (and the filename: `JsonUtils.h`).